static/app/components/autoSelectText.tsx

```diff
@@ -25,45 +25,40 @@ type Props = {
   style?: CSSProperties;
 };
 
-class AutoSelectText extends React.Component<Props> {
-  private el: HTMLElement | undefined;
+function AutoSelectText({children, className, ...props}: Props) {
```

Comment:
I wonder if this needs to become a `React.FC` to make those type errors bellow happy?